src/components/input-field/input-field.tsx

```diff
@@ -624,4 +625,17 @@ export class InputField {
             this.action.emit();
         }
     }
+    private handleWheel() {
+        if (this.type === 'number') {
+            // In some browsers (Chrome for example), hovering the input with
+            // the input focused, and scrolling, will both change the value
+            // AND scroll the page. We would prefer to never change the value
+            // on scroll, instead always scrolling the page, but since we
+            // haven't found a way to do that, this is the next best thing, as
+            // it prevents the page from being scrolled, but only in the
+            // circumstances when the value is changed by the scrolling.
+            // Please test THOROUGHLY if you remove it 😄
+            return true;
```

Comment:
What does this do? :thinking: I tried looking it up but didn't find anything

Reply:
It doesn't seem to do anything. The hack works regardless of whether we return `true`, `false`, or nothing. But since returning `false` from an event-listener used to mean approximately `preventDefault`, and we don't wan't to do that, I was thinking `return true` was probably the closest thing to something that makes sense… but maybe not having a return statement at all makes it clearer that it is the event-listener itself that seems to fix the problem, rather than what is returned from it…?

Reply:
Oh, by the way, there shouldn't be any need for the `if` either I guess… that's just left there from when we where doing `event.preventDefault` and stuff…

I'll experiment a little 😄

Reply:
Pushed a fixup.